addons/hr_holidays/views/hr_accrual_plan.xml

```diff
@@ -0,0 +1,88 @@
+<odoo>
```

Comment:
According to the guidelines, the file should be named ``hr_accrual_plan_views.xml``

+ Missing shebang.